contracts/test/IntegrationTestBase.sol

```diff
@@ -851,10 +851,8 @@ contract IntegrationTestBase is Test, TestParams, TestRegistry, TestSubnetActor,
             signatures[i] = abi.encodePacked(r, s, v);
         }
 
-        for (uint256 i = 0; i < n; i++) {
-            vm.prank(validators[i]);
-            saDiamond.checkpointer().submitCheckpoint(checkpoint, validators, signatures);
-        }
+        vm.prank(validators[0]);
```

Comment:
just like relayer, only need to submit once, not in a loop.

Reply:
Note that this is a digression from the previous model. Previously we would attempt to reward all relayers that submitted a checkpoint, although the reward path was unimplemented, and the overall concept somewhat questionable.